libraries/consul_service_windows.rb

```diff
@@ -64,12 +64,20 @@ def action_enable
       end
 
       def action_reload
-        Chef::Log.info 'The service provider for Consul on Windows does not support reload!'
+        notifying_block do
+          execute 'Reload consul' do
+            command 'consul.exe reload'
```

Comment:
Hi, 

No, it does not interface with NSSM in any way. I guess that you're referencing to [consul reload article](https://www.consul.io/docs/commands/reload.html) and use of SIGHUP for config reload. 

My interpretation is that if NSSM would catch this signal and actually restarted service, usual startup lines like ```==> Starting Consul agent...``` would be presented in the logs. That is not happening during my tests.